### PR TITLE
Fixes #1: Write README to explain the purpose of the repository and how to use the script.

### LINKED ISSUE #1: Write README to explain the purpose of the repository and how to use the script.
### Description :rocket:

As a new repository, current members and new members of CFIA will need to understand the purpose of the repository and how to use this tools to prepare their product iteration's report.

### Acceptance Criteria :white_check_mark:

- The README cover the purpose and the scope of the repository
- The README cover the usage of the script
- The README also propose avenue to improve the script

### Additional Context :pushpin:

This tools was develop to help developer save time to prepare their iteration report on their product project page.

image

### PR DESCRIPTION
**Pull request by MaxenceGui**
Fixes #1

- [x] Write the README to explain the purpose, the scope and the usage of the repository

### CODE REVIEW

**Review by amirardalan9473 — APPROVED** [review]

**Review by SonOfLope — APPROVED** [review]

**Review by elwoujdi — CHANGES_REQUESTED** [review]
the repository file_for_report doesn't exist in the workspace causing an error
```cmd
  File "C:\Users\Ibrahim Kabir\acia\report-automation\report_script.py", line 211, in <module>
    main()
  File "C:\Users\Ibrahim Kabir\acia\report-automation\report_script.py", line 207, in main
    start_programm()
  File "C:\Users\Ibrahim Kabir\acia\report-automation\report_script.py", line 200, in start_programm
    create_file_entry_frame("file_for_report")
  File "C:\Users\Ibrahim Kabir\acia\report-automation\report_script.py", line 168, in create_file_entry_frame
    files = os.listdir(directory)
```

File: README.md [hunk 1]
```diff
@@ -1,2 +1,66 @@
...
+You will see the following interface:
+
+![Script Interface](https://github.com/ai-cfia/report-automation/assets/96267006/27872abb-560c-4e37-988f-6af89a63974f)
+
+1. The right component allows you to add `.tsv` files. - Those files usually are
+  the current iteration that is ending and the milestones you want to showcase.
+   - You can collect those files from the project views. (Example Current
+  Iteration)
+1. The left component lets you enter important information such as `Author`
+   (your name), `Iteration Date`, and the `Milestones` you are working on.
```
Comment (elwoujdi) [inline README.md]: Add the format of iteration date (screenshot from github)

File: README.md [hunk 2]
```diff
@@ -1,2 +1,66 @@
...
+
+```bash
+python report_script.py
+```
+
+You will see the following interface:
+
+![Script Interface](https://github.com/ai-cfia/report-automation/assets/96267006/27872abb-560c-4e37-988f-6af89a63974f)
+
+1. The right component allows you to add `.tsv` files. - Those files usually are
```
Comment (elwoujdi) [inline README.md:32]: Add instructions by a gif, video, or bullet points

**Review by elwoujdi — COMMENTED** [review]
Tkinter is generating an error about the module tabulate. Add it to the requirements.txt
```cmd
$ python report_script.py
Exception in Tkinter callback
Traceback (most recent call last):
  File "C:\Users\Ibrahim Kabir\acia\report-automation\.venv\Lib\site-packages\pandas\compat\_optional.py", line 135, in import_optional_dependency
    module = importlib.import_module(name)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Ibrahim Kabir\.pyenv\pyenv-win\versions\3.11.1\Lib\importlib\__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen importlib._bootstrap>", line 1206, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1178, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1142, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'tabulate'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\Ibrahim Kabir\.pyenv\pyenv-win\versions\3.11.1\Lib\tkinter\__init__.py", line 1948, in __call__
    return self.func(*args)
           ^^^^^^^^^^^^^^^^
  File "C:\Users\Ibrahim Kabir\acia\report-automation\report_script.py", line 132, in generate_report
    generate_markdown("REPORT_TEMPLATE.md", f"{keys['date']}_report.md", "file_for_report", **keys)
  File "C:\Users\Ibrahim Kabir\acia\report-automation\report_script.py", line 25, in generate_markdown
    charts, markdown_table = prepare_file_for_report(directory)
                             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Ibrahim Kabir\acia\report-automation\report_script.py", line 64, in prepare_file_for_report
    table = df_sorted.to_markdown(index=False)
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Ibrahim Kabir\acia\report-automation\.venv\Lib\site-packages\pandas\util\_decorators.py", line 333, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Ibrahim Kabir\acia\report-automation\.venv\Lib\site-packages\pandas\core\frame.py", line 2983, in to_markdown
    tabulate = import_optional_dependency("tabulate")
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Ibrahim Kabir\acia\report-automation\.venv\Lib\site-packages\pandas\compat\_optional.py", line 138, in import_optional_dependency
    raise ImportError(msg)
ImportError: Missing optional dependency 'tabulate'.  Use pip or conda to install tabulate.
Traceback (most recent call last):
  File "C:\Users\Ibrahim Kabir\acia\report-automation\report_script.py", line 211, in <module>
    main()
  File "C:\Users\Ibrahim Kabir\acia\report-automation\report_script.py", line 207, in main
    start_programm()
  File "C:\Users\Ibrahim Kabir\acia\report-automation\report_script.py", line 204, in start_programm
    root.mainloop()
  File "C:\Users\Ibrahim Kabir\.pyenv\pyenv-win\versions\3.11.1\Lib\tkinter\__init__.py", line 1485, in mainloop
    self.tk.mainloop(n)
```

**Review by elwoujdi — COMMENTED** [review]
image

Replace to the project, not only Nachet.

image

Add maybe a checkbox to enable or disable milestones

Finally maybe add an output folder

File: README.md (hunk 2 above)
Comment (MaxenceGui) [reply README.md:32]: Done

File: README.md (hunk 1 above)
Comment (MaxenceGui) [reply README.md]: Done

**Comment by MaxenceGui** [comment]
I think I fixed and implement everything you point out @ibrahim-kabir

**Review by elwoujdi — APPROVED** [review]
LGTM

**Pull request merged**